src/enn_undersample/interface_mlj.jl

```diff
@@ -0,0 +1,164 @@
+
+### ENNUndersampler with MLJ Interface
+# interface struct
+mutable struct ENNUndersampler{T} <: Static
+    k::Integer
+    keep_condition::AbstractString
+    min_ratios::T
+    force_min_ratios::Bool
+    rng::Integer
```

Comment:
It looks to me that `AbstractRNG` is also supported by your implementation. See line 170 of src/enn_undersample/enn_undersample.jl .